packages/@okta/vuepress-site/guides/sign-into-spa/section07/react/login-redirect.md

```diff
@@ -1,22 +1,11 @@
-
-The `Auth` object is provided on the component's props via the `withAuth` Higher Order Component (HOC).
-You can show/hide the correct button by checking the value returned from the `auth.isAuthenticated()` method. 
+The `Auth` object is provided on your component's props via the `withAuth` component. You can use the `auth.isAuthenticated()` method to show or hide a button depending on whether the user is signed in.
 
 ```javascript
```

Comment:
@aarongranick-okta Same comment as earlier - it seems like a reader could lose track of "which file should I copy this into". I realize we don't know what their file structure looks like, but do you think we could give them a hint?

Reply:
Outside our samples, we can't really say. Many apps would place the login/logout button in a common header component. But it may be presented on a landing page. In our sample, we do both.